client/src/containers/ListEvents.js

```diff
@@ -1,128 +1,133 @@
 // @flow
 
 import React, { Component } from 'react';
-import {
-    Row,
-    Col,
-    Jumbotron,
-    Button
-  } from 'reactstrap';
+import { Row, Col, Jumbotron, Button } from 'reactstrap';
 import { userLogin, fetchEvents, postEvent, userLogout } from '../actions';
 import { Action } from 'redux';
 import type { BaseReduxPropTypes } from '../types/base-props-types';
 import { connect } from 'react-redux';
+import { EventList } from '../components/EventList/EventList';
 
 type Props = BaseReduxPropTypes & {
-	userState: Object,
-	appState: Object,
-	events: Object
-}
+  userState: Object,
+  appState: Object,
+  events: Object,
+};
+const eventsDummy = [
+  {
+    key: '1',
+    title: 'Road to Angular',
+    location: 'Karachi',
+    startDateTime: '14-July-2018',
+    endDateTime: '21-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
+    organisation: 'ngGirls',
+  },
+  {
+    key: '2',
+    title: 'Google I/O',
+    location: 'Lahore',
+    startDateTime: '28-July-2018',
+    endDateTime: '29-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
+    organisation: 'GDG Kolachi',
+  },
+  {
+    key: '3',
+    title: 'Road to JavaScript',
+    location: 'Islamabad',
+    startDateTime: '14-July-2018',
+    endDateTime: '21-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
+    organisation: 'ngGirls',
+  },
+  {
+    key: '4',
+    title: 'Google Devfest',
+    location: 'Italy',
+    startDateTime: '28-July-2018',
+    endDateTime: '29-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
+    organisation: 'GDG Kolachi',
+  },
+  {
+    key: '5',
+    title: 'Node School Workshop',
+    location: 'Malaysia',
+    startDateTime: '28-July-2018',
+    endDateTime: '29-July-2018',
+    imageurl: 'https://cdn-images-1.medium.com/max/800/0*PkFv7ExBsCL5xrHA.',
```

Comment:
Please check the model in the documentation, I think this is named cover.

Reply:
@mashhoodr we need attribute for location in the model

Reply:
@mashhoodr , there is a suggestion for event model based on what we can have in event details page. we can further refine it.
Event Model:
- id
- title
- description
- cover
- starttime
- endtime
- ticketurl
- organization
	- name
	- logo
	- about
	- contact_details
- organizers
	- name
	- email
	- contact_num
- location
	- venue name
	- map location
- Sponsors